e2e/app/aou-elements/button.ts

```diff
@@ -27,12 +27,13 @@ export default class Button extends BaseElement {
   /**
    * Wait until button is clickable (enabled).
    */
-  async waitUntilEnabled(): Promise<JSHandle> {
-    const handle = this.element.asElement();
-    return await handle.evaluateHandle((e) => {
+  async waitUntilEnabled(): Promise<unknown> {
+    const result = await this.page.waitForFunction((e) => {
       const style = window.getComputedStyle(e);
       return style.getPropertyValue('cursor') === 'pointer';
-    }, this.element);
+    }, {}, this.element);
+    const jval = await result.jsonValue();
+    return jval;
```

Comment:
Request `GET /v1/institutions/Broad/checkEmail/aou-dev-registration@broadinstitute.org/` could takes up to several seconds to complete. Changed to `waitForFunction` for waiting function returns true, up to 30 seconds.